src/components/markdown-renderer/replace-components/markmap/markmap-frame.tsx

```diff
@@ -57,7 +57,7 @@ export const MarkmapFrame: React.FC<MarkmapFrameProps> = ({ code }) => {
         console.error(error)
       }
     })
-                              .catch(() => {
+                                                               .catch(() => {
```

Comment:
Technically that's correct due to our editor config.
What about moving the `.then()` after the `import` into the next line to have the `.catch` not intended that much?

Reply:
fixed